chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/surveillance/NonconformityTypeStatisticsEntity.java

```diff
@@ -0,0 +1,97 @@
+package gov.healthit.chpl.entity.surveillance;
+
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "nonconformity_type_statistics")
+public class NonconformityTypeStatisticsEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+    
+    @Basic(optional = false)
+    @Column(name = "nonconformity_count", nullable = false)
+    private Long nonconformityCount;
+    
+    @Basic(optional = false)
+    @Column(name = "nonconformity_type", nullable = false)
+    private String nonconformityType;
```

Comment:
What are the options for nonconformityType? Should this be an enum, or a foreign key to some other table?

Reply:
The nonconformityType in this table is the Criteria name, might be better if it was called criteriaName or something.

Reply:
Is it the same nonconformity_type field as is in the surveillance_nonconformity table? If so, it can be either a criteria name or 170.523 (k)(1) or (k)(2) which are not criteria.

Reply:
In that other table it's a varchar of some sort, right? My guess is that we should change it to an enum or lookup at some point, but for now keeping this table and that table in sync as both being Strings is fine